package/timower-stuff/package

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(yaft tilem)
```

Comment:
This should be split into two separate packages that can be updated independently.

Reply:
I'm fine either way, but they'll always be updated together as they're part of the same repo with the same tags.
So to simplify the build process I made them one package.

Reply:
i think it's reasonable to keep as is (one package file with multiple packages out of the same repo), but that's because how i also do it